build.gradle

```diff
@@ -1,35 +1,22 @@
-plugins {
-    id 'org.jetbrains.kotlin.jvm' version "${kotlinJvm}"
-    id "org.jlleitschuh.gradle.ktlint" version "${ktlintGradle}"
-}
-
-group = 'org.dif'
-version = "${version}"
-
-repositories {
-    mavenCentral()
-    flatDir {
-        // TODO remove when contribution will be merged to upstream
-        dir "lib"
+// Top-level build file where you can add configuration options common to all sub-projects/modules.
+buildscript {
+    repositories {
+        google()
```

Comment:
Are it needed? I thought you move repositories to settings.gradle, please correct me If i'm not right.

Reply:
can't answer for sure here, but I thought the same and it was wrong: by some reason it fails if the repos are not listed here as well, I suggest to dig into that later one more time